### PR TITLE
newgrp: add page

### PR DESCRIPTION
**Pull request by lbonanomi**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by lbonanomi** [comment]
This PR is in support of #2214

File: pages/linux/newgrp.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# newgrp
+
+> Switch primary group membership.
```
Comment (sbrl) [inline pages/linux/newgrp.md]: Of who or what?

Comment (lbonanomi) [reply pages/linux/newgrp.md]: @sbrl: is this a job for a fresh PR or can master still be restored?

Comment (sbrl) [reply pages/linux/newgrp.md]: I think a fresh PR is needed here, yeah

**Pull request merged**